torch/_torch_docs.py

```diff
@@ -3856,6 +3856,73 @@ def merge_dicts(*dicts):
     torch.return_types.median(values=tensor([-0.3982,  0.2270,  0.2488,  0.4742]), indices=tensor([1, 4, 4, 3]))
 """.format(**single_dim_common))
 
+add_docstr(torch.quantile,
+           r"""
+quantile(input, q) -> Tensor
+
+Returns the quantile values of all elements in the :attr:`input` tensor.
```

Comment:
Can you relate this to `q`? The NumPy documentation says the "q-th quantile," which is better, but fails to cover the case when q is a tensor.

Reply:
Also: should this describe the linear interpolation between elements?

Reply:
(You have the note below, but not here, and it seems more like regular documentation than a note, anyway.)